src/command/mcl.zig

```diff
@@ -10,6 +10,35 @@ var line_accelerations: []u7 = undefined;
 
 const Direction = mcl.Direction;
 const Station = mcl.Station;
+// const Registers = struct {
+//     x: mcl.registers.X,
+//     y: mcl.registers.Y,
+//     wr: mcl.registers.Wr,
+//     ww: mcl.registers.Ww,
+// };
+
+// const RegisterType = enum { x, y, wr, ww };
+// var register_log_file: ?std.fs.File = null;
+// var register_list = std.EnumArray(RegisterType, bool).initFill(false);
+// var station_id_log: std.ArrayList(u8) = undefined;
+
+const RegisterLogging = struct {
+    line_name: []const u8,
+    register_list: std.EnumArray(RegisterType, bool), // Specify which register to log for each line
+    station_id: []u8, // Specify the station to log based on axis provided by user
+    station_len: u8, // Specify the number of stations to be logged
```

Comment:
Why are we storing this separately? If we are using a slice of station IDs, the slice already contains the number of stations in its length field.